src/api/app/controllers/webui/package_controller.rb

```diff
@@ -120,7 +120,11 @@ def update
           flash[:success] = 'Package was successfully updated.'
           redirect_to package_show_path(project: @project, package: @package)
         end
-        format.js { flash.now[:success] = 'Package was successfully updated.' }
+        format.js do
+          @report_bug_or_bugzilla_url = report_bug_or_bugzilla_url
```

Comment:
Why don't you set this outside of the `format` block? 

BTW how does one reach the non-js version of this controller action anyway?

Reply:
> Why don't you set this outside of the format block?

Setting this outside of the format block will affect the `format.html` block too. The `format.html` block doesn't make use of this variable.

Reply:
Sure but is this even used? :)

Reply:
> BTW how does one reach the non-js version of this controller action anyway?

No idea. I couldn't reach this code with my tests. This doesn't belong to the changes I proposed. We could address this on a follow-up pull request.